### DISCUSSION TITLE
ungroup same program taskbar buttons

### DISCUSSION THREAD

**Discussion by fleadhfan** [valinet/ExplorerPatcher#3725] [Ideas]
Is this possible, either with an EP enhancement, registry key, or any other method? I often have multiple program instances open, but each is related to different other overall tasks I'm doing using multiple programs, and I'd like to be able to position the buttons together by my tasks, and not have all buttons for a given program grouped together just because Microsoft thinks that's how it should be. Thanks!

**Comment by pyrates999** [comment]
try this with the windows 11 taskbar for ungrouping taskbar icons: https://www.prajwaldesai.com/never-combine-taskbar-buttons-in-windows-11/

you might also want to try fancy zones: https://learn.microsoft.com/en-us/windows/powertoys/

you can also create multiple desktops: https://support.microsoft.com/en-us/windows/multiple-desktops-in-windows-36f52e38-5b4a-557b-2ff9-e1a60c976434#WindowsVersion=Windows_11

and for the Windows 10 (ExplorerPatcher) in windows 11: https://github.com/valinet/ExplorerPatcher/wiki/All-features#combine-taskbar-icons-on-primarysecondary-taskbars

**Comment by fleadhfan** [comment]
Ok, found a solution that works the way I was hoping. I was already using ExplorerPatcher. Added the also free 7+ Taskbar Tweaker and changed its Grouping option to "Don't Group". This allows individual buttons of the same program to be positioned anywhere on the taskbar.

**Reply by watashigoto** [reply]
I was forced to update to 24H2.
Taskbar Tweaker no longer works there, you need to install additional Windhawk program to make it works again.
Sad that feature request for this is not popular 😭

**Reply by LiivaneLord** [reply]
> I was forced to update to 24H2. Taskbar Tweaker no longer works there, you need to install additional Windhawk program to make it works again. Sad that feature request for this is not popular 😭

I have forced myself to destroy 24H2 and return to 23H2.
It is so damn good that all taskbar items can be ungrouped and I also love to set taskbar at the left side of screen. I cant understand why it is so hard for Microsoft to make taskbar much-much more comfortable.
Also it is so damn good to close programs by mouse middle click and minimize/restore with mouse wheel (this one took a while to be familiar with).

Waiting for explorer patcher and 7+ tweaker update to use newer version of windows. If these updates not coming then I feel like I have to stay with 23H2 forever. :(
